applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -373,6 +373,61 @@ config BT_AUDIO_BROADCAST_IMMEDIATE_FLAG
 	help
 	  Set the immediate rendering flag.
 
+config BT_AUDIO_PREFERRED_MIN_PRES_DLY_US
+	int "The preferred minimum presentation delay"
+	range AUDIO_MIN_PRES_DLY_US AUDIO_MAX_PRES_DLY_US
+	default 10000
+	help
+	  The preferred minimum presentation delay in micro seconds. This can not
+	  be less than the absolute minimum presentation delay.
+
+config BT_AUDIO_PREFERRED_MAX_PRES_DLY_US
+	int "The preferred maximum presentation delay"
+	range AUDIO_MIN_PRES_DLY_US AUDIO_MAX_PRES_DLY_US
+	default 40000
+	help
+	  The preferred maximum presentation delay in micro seconds. This can not
+	  be less than the absolute maximum presentation delay.
+
+choice	BT_AUDIO_PRES_DLY_SRCH
+	prompt "Default search mode for the presentation delay"
+	default BT_AUDIO_PRES_DELAY_SRCH_LOCAL
+	help
+	  Set the default search mode for the presentation delay.
+
+config BT_AUDIO_PRES_DELAY_SRCH_MIN
+	bool "Largest minimum delay over all headsets"
+	help
+	  Search for the largest minimum delay over all headsets.
+
+config BT_AUDIO_PRES_DELAY_SRCH_MAX
+	bool "Smallest maximum delay over all headsets"
+	help
+	  Search for the smallest maximum delay over all headsets
+
+config BT_AUDIO_PRES_DELAY_SRCH_PREF_MIN
+	bool "Search for the largest minimm preferred delay over all headsets"
```

Comment:
True, but I thought this would confuse as currently we refer to all audio receivers as headsets. So not keen to mix definitions until the rework of naming is performed in its own PR.